torch/csrc/api/include/torch/nn/modules/container/any.h

```diff
@@ -285,19 +285,11 @@ class AnyModule::Value {
   friend struct TestValue;
 
   /// Constructs the `Value` from value type.
-  template <
-      typename T,
-      typename =
-          torch::disable_if_t<std::is_same<autograd::Variable, T>::value>>
+  template <typename T>
```

Comment:
I'm not sure what the original code was doing (special case for Variable?). Is the point that Variables don't exist so this special case is not needed anymore?

Reply:
Well, the old code disabled the template if you passed in a Variable, so only Tensors OK. That's because they had an explicit overload for the Variable case, and didn't want the template to interfere with it.